xml/deployment_prep_aarch64_raspi.xml

```diff
@@ -0,0 +1,239 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE sect1
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+
+<sect1 version="5.0" xml:id="sec.aarch64.raspi"
+ xmlns="http://docbook.org/ns/docbook"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink">
+ <title>&rpi;</title>
+
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+
+ <para>
+  &slsreg; is the first enterprise Linux distribution to fully support the
+  inexpensive &rpi; single-board computer. &sls; 15.1 supports only the &rpi; 3
+  Model B and B+.
+ </para>
+
+ <para>
+  There are two different ways to deploy &sls; onto &rpi; hardware:
+ </para>
+ <itemizedlist>
+  <listitem>
+   <para>
+    Copy a pre-installed system image onto a MicroSD card.
+   </para>
+   <para>
+    This is the most common method for deploying operating systems on the
+    &rpi; platform.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    Boot from USB media and install onto another drive.
+   </para>
+   <para>
+    This is the more conventional way to install &sls;, although it is less
+    common for &rpi; operating systems.
+   </para>
+  </listitem>
+ </itemizedlist>
+ 
+ <para>
+  There are several points to note regarding both methods.
+ </para>
+ 
+ <sect2>
+  <title>Deploying from a Preinstalled Image</title>
+  <para>
+   This method is the simplest and easiest method. However, it allows little
+   customisation of the product being deployed except for values such as network
```

Comment:
values?

Reply:
Yes, values. Would you prefer the word "settings"?